### PR TITLE
Make rate limits clearer

### PR DESCRIPTION
**Pull request by quis**
This commit proposes:
- tidying up the rate limit section so that it’s clearer and less wordy
- front loading the important information (the 3000 messages/min)
- adding some info about rate limits imposed by the phone networks, because people keep asking why their messages have started failing to be delivered

### CODE REVIEW

File: source/documentation/_limits.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
 # Limits
 
-## API rate limits
+## Rate limits
 
-There is a rate limit for requests to the API from your account. The limit is 3000 per rolling 60 second time period. This limit applies per API key type. If you exceed the limit, you will receive the `429` error `RateLimitError`.
+You’re limited to sending 3000 messages per minute.
```
Comment (karlchillmaid) [inline source/documentation/_limits.md]: Style guide says use a comma for numerals over 999 for clarity: `3,000`

File: source/documentation/_limits.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 
-## Service limits
+This limit is calculated on a rolling basis, per API key type. If you exceed the limit, you will get a `429` error `RateLimitError`.
 
-There are limits for the number of messages sent per day. These limits depend on whether your service is live, and the API key type. Limits reset at midnight.
+## Daily limits
 
-|Service status|API key type|Daily limit|
+There’s a limit to how many messages you can send each day:
```
[the remark this answers is not in the document]
Comment (jonathanglassman) [reply source/documentation/_limits.md]: Agree with karl's feedback

**Comment by quis** [comment]
@karlchillmaid @jonathanglassman we got an answer back from one of the providers. I’ve added this info in 32eac527c77649e68d8596608f2804fe5db9fca8. Feel free to suggest changes if you think it could be clearer.

**Comment by quis** [comment]
@karlchillmaid nice – updated in d8973c8

**Pull request merged**